src/constants/bot-messages.js

```diff
@@ -0,0 +1,17 @@
+export const invalidCommand = "Error! Error! Invalid command!";
+export const missingArgs = "Missing command arguments, Please contact FatBoy if you are a female he will help you :)";
+export const executionError = "An error occurred while executing the command :(";
+export const cannotEdit = "You cannot edit that message retard.";
+export const specifyMessageId = "Please specify the message id and the modified message, freakin scrub.";
+export const messageDoesNotExist = " Message does not exist, may have been deleted or not sent at all.";
```

Comment:
Just about naming the constants. Shall we have them in caps so that it implies that this a constant which should not be changed.

```suggestion
export const INVALID_COMMAND = "Error! Error! Invalid command!";
```